bower.json

```diff
@@ -8,6 +8,7 @@
   "main": "quick-element.html",
   "license": "MIT",
   "dependencies": {
-    "polymer": "polymer/polymer#0.8-preview"
+    "polymer": "polymer/polymer#0.8-preview",
+    "quick-element": "polymer/quick-element"
```

Comment:
When you install `quick-element` as dependency it either is already installed and skipped - that's the case when using quick-element as dependency of another project - or it's used without a context. In this case also polymer gets installed to `bower_components` within the working directory.

That allows the link href in `quick-element.html` to point to the right location. Indeed, this is not the nicest solution, but it only adds overhead when using the demo or checking out `quick-element` for development.